.github/workflows/code-coverage-linux.yml

```diff
@@ -1,4 +1,4 @@
-name: CI Pipeline Code Coverage Linux
+name: Coverage Linux
```

Comment:
Suggestion to name this "Code Coverage Linux" to make it clear what _kind_ of coverage will be tested.

Reply:
There is nothing else to cover..... I needed smaller names for the badges and this explains perfectly well what this action does

Reply:
Ah, I didn't know this showed on the badges as well. Then it makes perfect sense to keep them as short as possible!